connectivity/FEATURE_BLE/include/ble/gatt/GattCallbackParamTypes.h

```diff
@@ -443,8 +443,31 @@ struct GattDataSentCallbackParams {
 
 };
 
-using GattUpdatesEnabledCallbackParams        = GattDataSentCallbackParams;
-using GattUpdatesDisabledCallbackParams       = GattDataSentCallbackParams;
+/**
+ * Gatt Updates Enabled Attribute related events
+ *
+ * Used by `onUpdatesEnabled`
+ */
+struct GattUpdatesEnabledCallbackParams {
```

Comment:
Maybe use a name applicable to both updates enabled and disabled then alias the type: 

```c++
struct GattUpdatesChangedCallbackParams { };

using GattUpdatesEnabledCallbackParams = GattUpdatesChangedCallbackParams;
using GattUpdatesDisabledCallbackParams = GattUpdatesChangedCallbackParams;
```

Reply:
I agree: aaf9435f87b9294e82e206085cd475b1bd6c28dd. However, the same can be said for `GattDataSentCallbackParams`.